typings/lhr-lite.d.ts

```diff
@@ -0,0 +1,132 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+declare global {
+  module LH {
+    /**
+     * The lightweight version of Lighthouse results.
+     */
+    export interface ResultLite {
+      /** The URL that was supplied to Lighthouse and initially navigated to. */
+      requestedUrl: string;
+      /** The post-redirects URL that Lighthouse loaded. */
+      finalUrl: string;
+      /** The ISO-8601 timestamp of when the results were generated. */
+      fetchedAt: string;
+      /** The version of Lighthouse with which these results were generated. */
+      lighthouseVersion: string;
+      /** An object containing the results of the audits. */
+      audits: Record<string, ResultLite.Audit>;
+      /** An object containing the top-level categories, their overall scores, and reference to member audits. */
+      categories?: Record<string, ResultLite.Category>;
+      /** Descriptions of the audit groups referenced by AuditRefs. */
+      categoryGroups?: Record<string, ResultLite.CategoryGroup>;
```

Comment:
Did we decide of `groups` for this? I don't remember at this point 😆 or `auditGroups` aka "groups of related audits"